### ISSUE TITLE
Using NFS as remote artifact store with Windows client

### ISSUE THREAD

**Issue by swigicat** [mlflow/mlflow#3476]
Hi,

This seems to be a popular topic and I've read through a lot of tickets, but haven't found an answer, so I am not sure if this is impossible or just tricky to set up. 

My use-case is that I have a Windows computer that runs some experiments locally, then tries to log metrics and artifacts to a remote server (Linux). The metrics are stored and work very well. However, it's not clear how to set up the remote artifact store using NFS. The documentation states that the paths must be identical on the client and the server, but I'm not sure how this can be done in Windows - everything has backspaces or has drive letters in the path. Can this be done right now?

**Comment by dmatrix** [comment]
@swigicat Yes, both the client and where the remote tracking server. So if you have mounted your Linux-based NFS on your windows machine d:\\mnt\tracking and on the Linux tracking server as /mnt/tracking, it should work.

On the linux machine, launch your tracking server:

```
   mlflow server \
    --backend-store-uri /mnt/registery.db \
    --default-artifact-root /mnt/tracking \
    --host 0.0.0.0
```

On. your client windows machine in the code:

`mlflow.set_tracking_uri("http://ip_of_tracking_server:5000")
`

Does this work for you?

**Comment by ramchinthamsetty** [comment]
Hi, I am running the same scenario. My server running on Linux with NFS share and I am running the mlflow runs on Windows. Now, I have mounted the NFS share as X: drive and on Linux it was mounted on /mnt/artifactory. 
mlflow server --backend-store-uri sqlite:///mydb.sqlite --default-artifact-root /mnt/artifactory --host 0.0.0.0 -p 9001

But, still, I wasn't able to log the artifacts to the Linux server NFS share. 

Would you please help with this issue?
